resources/default_pipeline_environment.yml

```diff
@@ -219,6 +219,13 @@ steps:
       languageRunner: 'js'
       runCommand: 'gauge run'
       testOptions: 'specs'
+    bundler:
+      dockerImage: 'ruby:2.5.3-stretch'
+      dockerName: 'bundler'
+      dockerWorkspace: '/home/seluser'
```

Comment:
we need to use the workspace in the ruby image, I guess `/home/seluser` is not existing there.